### PR TITLE
[PaddlePaddle Hackathon] add ShuffleNetV2

### PR DESCRIPTION
**Pull request by SigureMo**
### PR types

New features

### PR changes

APIs 

### Describe

- Task: https://github.com/PaddlePaddle/Paddle/issues/35996
- 向 paddle.vision.models 添加以下模型

  - ShuffleNetV2
  - shufflenet_v2_x0_25
  - shufflenet_v2_x0_33
  - shufflenet_v2_x0_5
  - shufflenet_v2_x1_0
  - shufflenet_v2_x1_5
  - shufflenet_v2_x2_0
  - shufflenet_v2_swish

status: pending review

- [x] 支持预训练模型
- [x] 单元测试
- [x] 文档 https://github.com/PaddlePaddle/docs/pull/3924

### Performance

AiStudio 测试详情：<https://aistudio.baidu.com/aistudio/projectdetail/2420498>
基准参考：<https://github.com/PaddlePaddle/PaddleClas/blob/release/2.2/docs/en/ImageNet_models_en.md>

|Model|Top-1|Top-5|
|-|-|-|
|ShuffleNetV2_x0_25|0.4998<span style="color:red;"><sub>(+0.0008)</sub></span>|0.7373<span style="color:green;"><sub>(-0.0006)</sub></span>|
|ShuffleNetV2_x0_33|0.5357<span style="color:green;"><sub>(-0.0016)</sub></span>|0.7706<span style="color:red;"><sub>(+0.0001)</sub></span>|
|ShuffleNetV2_x0_5|0.6033<span style="color:red;"><sub>(+0.0001)</sub></span>|0.8224<span style="color:green;"><sub>(-0.0002)</sub></span>|
|ShuffleNetV2_x1_0|0.6879<span style="color:green;"><sub>(-0.0056)</sub></span>|0.8844<span style="color:red;"><sub>(+0.0012)</sub></span>|
|ShuffleNetV2_x1_5|0.7157<span style="color:green;"><sub>(-0.0006)</sub></span>|0.9013<span style="color:green;"><sub>(-0.0002)</sub></span>|
|ShuffleNetV2_x2_0|0.7314<span style="color:green;"><sub>(-0.0001)</sub></span>|0.9123<span style="color:red;"><sub>(+0.0003)</sub></span>|
|ShuffleNetV2_swish|0.7008<span style="color:red;"><sub>(+0.0005)</sub></span>|0.8914<span style="color:green;"><sub>(-0.0003)</sub></span>|

括号中为以 PaddleClas 性能基准为参考的偏差值，参考中无 ShuffleNetV2_x1_0 模型，其中对比是与 PyTorch（<https://pytorch.org/vision/stable/models.html>）中的基准进行对比的

### CODE REVIEW

**Review by LielinJiang — DISMISSED** [review]

**Comment by SigureMo** [comment]
@LielinJiang 刚刚解决了冲突并通过了 CI，可否麻烦再次 review 一下呀～

File: python/paddle/vision/models/shufflenetv2.py [hunk 1]
```diff
@@ -0,0 +1,480 @@
+# Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```
Comment (LielinJiang) [inline python/paddle/vision/models/shufflenetv2.py]: 2021

Comment (SigureMo) [reply python/paddle/vision/models/shufflenetv2.py]: 已修改～～～

**Comment by SigureMo** [comment]
@LielinJiang 可以麻烦再 review 一下这个嘛òᆺó？

**Review by LielinJiang — APPROVED** [review]

**Review by TCChenlong — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LG API

**Comment by SigureMo** [comment]
@TCChenlong CI 过啦 :tada: ～～～

**Comment by SigureMo** [comment]
喔喔对了，任务总览里这个还木有更新～另外辛苦各位这段时间的 review 啦～

**Pull request merged**